website/settings/defaults.py

```diff
@@ -345,15 +345,62 @@ def parent_dir(path):
 WATERBUTLER_INTERNAL_URL = WATERBUTLER_URL
 WATERBUTLER_ADDRS = ['127.0.0.1']
 
-# Test identifier namespaces
-DOI_NAMESPACE = 'doi:10.5072/FK2'
-ARK_NAMESPACE = 'ark:99999/fk4'
+# Set which client to use for which type of object
+PREPRINT_DOI_CLIENT = 'crossref'  # set to 'ezid' to use old ezid client
+NODE_DOI_CLIENT = 'ezid'
 
-# For creating DOIs and ARKs through the EZID service
+# Test EZID identifier namespaces
+EZID_DOI_NAMESPACE = 'doi:10.5072'
+EZID_ARK_NAMESPACE = 'ark:99999'
+
+# For creating DOIs and ARKs
 EZID_USERNAME = None
 EZID_PASSWORD = None
-# Format for DOIs and ARKs
-EZID_FORMAT = '{namespace}osf.io/{guid}'
+
+CROSSREF_USERNAME = None
+CROSSREF_PASSWORD = None
+CROSSREF_DEPOSIT_URL = None
+CROSSREF_DEPOSITOR_EMAIL = 'None'  # changeme in local.py
```

Comment:
This will be whatever email we want to configure with mailgun to receive the confirmation/error messages! The thought was something like `crossref@cos.io` but thought it'd be nice to have it be configurable locally to test with mailgun locally.  I'll add a comment for clarity!